drivers/amazonec2/amazonec2.go

```diff
@@ -126,8 +126,8 @@ type Driver struct {
 	DisableSSL              bool
 	UserDataFile            string
 	EncryptEbsVolume        bool
-
-	spotInstanceRequestId string
+	spotInstanceRequestId   string
+	KmsKeyId                string
```

Comment:
no need to export this, should be lower case

Reply:
Addressed this.